src/modules/app/route-configs/MainDisclaimerRouteConfig.js

```diff
@@ -2,11 +2,14 @@
 
 import { RouteConfig } from './RouteConfig'
 
+// This type should be exact, but flow has a bug preventing this: https://github.com/facebook/flow/issues/2977
```

Comment:
Try to use an exact type here and use $FlowFixMe to supress the error.

Reply:
Done.